go.mod

```diff
@@ -113,9 +114,12 @@ replace (
 	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.6
 	k8s.io/code-generator => k8s.io/code-generator v0.23.6
 	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.23.6
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf
 	k8s.io/kubectl => k8s.io/kubectl v0.23.6
 	k8s.io/kubelet => k8s.io/kubelet v0.23.6
 	k8s.io/metrics => k8s.io/metrics v0.23.6
+	k8s.io/utils => k8s.io/utils v0.0.0-20211116205334-6203023598ed
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.11.2
```

Comment:
Why update this to 0.12 and then override it with 0.11.2 again?

Reply:
We have to pin it because it gets updated to `0.12` because of https://github.com/kubermatic/machine-controller/blob/master/go.mod#L56